test/test_type_promotion.py

```diff
@@ -138,39 +139,39 @@ def test_mixed_type_backward(self):
         f_dtypes = [torch.float, torch.double]
         f_dtypes = f_dtypes if self.device == 'cpu' else f_dtypes + [torch.half]
         i_dtypes = [torch.int, torch.long]
-        for f in ['add', 'sub', 'rsub', 'mul', 'div']:
+        for func in [torch.add, torch.sub, torch.rsub, torch.mul, torch.div]:
```

Comment:
Nit: itertools.product can save levels of indentation

Reply:
removed one level